### ISSUE TITLE
No Content Available (Only Shows Media When Inside Library)

### ISSUE THREAD

**Issue by zluckytraveler** [pannal/plex-for-kodi#118]
## Description:
Media not showing under Library when highlighted on Library, however if I select a library it will then display my media titles. Additionally only “Continue Watching” media will be displayed for TV libraries when highlighted over them.

I get the following message…
`Please add content and/or check that ‘Include in dashboard’ is enabled`

Where is the “Include in Dashboard” setting? 
Why is the media not being displayed? 

Im also not sure if this is correct, but when highlighted on Home it only displays “Continue Watching” and nothing else besides that.

Basically anything on the Home Screen when highlighted will only show “Continue Watching” media. 

## Details:
- Add-on version: 0.7.8
- OS Platform: Android TV
- Hardware: Nvidia Shield TV Pro

**Comment by pannal** [comment]
> Where is the “Include in Dashboard” setting?

Plex Web -> Settings -> Libraries:
![image](https://github.com/pannal/plex-for-kodi/assets/1359593/52186891-3ec5-451a-b1f3-8763113001a2)

**Comment by zluckytraveler** [comment]
> > Where is the “Include in Dashboard” setting?
> 
> Plex Web -> Settings -> Libraries: ![image](https://private-user-images.githubusercontent.com/1359593/341997327-52186891-3ec5-451a-b1f3-8763113001a2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkxMTI5MDUsIm5iZiI6MTcxOTExMjYwNSwicGF0aCI6Ii8xMzU5NTkzLzM0MTk5NzMyNy01MjE4Njg5MS0zZWM1LTQ1MWEtYjFmMy04NzYzMTEzMDAxYTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjNUMDMxNjQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmMwMGM1YzlmNGIxZjZkNTA5ZWU0OTc5ZmFmODdkNDhmMzkyYTkzODMzYzVjMDIzMzcyMjM4ODJjNjBkMzc1ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.WEkebtUWdYyJ6zop2QznEycxEu2lVsy_ltca0Hi__vs)

Yup those collections are definitely enabled for both Library & Home Screen . However I’m still seeing absolutely nothing! 

I don’t even have a watchlist or discover tab available.

**Comment by pannal** [comment]
> I don’t even have a watchlist or discover tab available.

Neither of those are available in PM4K right now.

Please share a screenshot.
